survival_guide/3_operations.adoc

```diff
@@ -590,32 +590,45 @@ Our recommended configuration for using the Spark History Server with Fusion is
 [source,yaml]
 ----
 gcs:
-    enableGCS: false
-    secret: history-secrets
-    key: [SECRET_KEY_NAME].json
-    logDirectory: gs://[BUCKET_NAME]
+  enableGCS: true
+  secret: history-secrets
+  key: sparkhistory.json
+  logDirectory: gs://[BUCKET_NAME]
 service:
-    type: ClusterIP
-    port: 18080
+  type: ClusterIP
+  port: 18080
 ----
-We override the default `service.type` of `LoadBalancer` with `ClusterIP` to keep the History Server from being accessible to outside connections without port-forwarding.
+The `service` key sets up the history server on an internal IP within your cluster but does not create an external LoadBalancer (which is the default setting in the Spark History Server Helm chart). This prevents the server from being exposed to outside access by default. We’ll look at how to access the history server shortly.
```

Comment:
Don't need "by default", it confuses our recommended default (which isn't really a default, it's just our recommendation) with the spark history server default in their helm chart. Also reduced the needed context for each sentence. And we mark this as a "note" because it's not essential to understand this in order to follow the steps.
```suggestion
Note that, by default, the Spark History Server Helm chart creates an external LoadBalancer, exposing it to outside access. This is usually undesirable. In the above, we prevent this via the `service` key - the history server will only be set up on an internal IP within your cluster and will not be exposed externally. Later, we will show how to properly access the history server.
```